rasterio/_base.pyx

```diff
@@ -677,8 +676,6 @@ cdef class DatasetBase(object):
         """
         def __get__(self):
             m = Profile(**self.meta)
-            m.update((k, v.lower()) for k, v in self.tags(
-                ns='rio_creation_kwds').items())
```

Comment:
This is the heart of the matter: we no longer marshal tags – which may not be truthful – into the dataset profile.